chase/chase.py

```diff
@@ -2,26 +2,67 @@
 from uuid import uuid4
 import xml.etree.ElementTree as ET
 import requests
-import re
 import unicodedata
-from time import sleep
+
 
 PROCSTATUS_INVALID_RETRY_TRACE = '9714'
 PROCSTATUS_USER_NOT_FOUND = '9581'
 
-CARD_TYPE_VISA = 'VISA'
+ELECTRONIC_CHECK = 'EC'
```

Comment:
beyond the scope of the refactor, but we should probably move these to a `config.py`